addons/binding/org.openhab.binding.amazonechocontrol/README.md

```diff
@@ -0,0 +1,303 @@
+# Amazon Echo Control Binding
+
+This binding can control Amazon Echo devices (Alexa) from openhab.
+
+It provide features to control and view the current state of echo devices:
+
+- volume
+- pause/continue/next track/previous track
+- connect/disconnect bluetooth devices
+- start playing tuneIn radio
+- start playing Amazon Music
+- control of multi room music
+- show album art image in sitemap
+- speak a reminder message
+- plays an alarm sound
+- start traffic news
+- start daily briefing
+- start weather report
+- start good moring report
+- start automation routine
+- activate multiple configurations of flash briefings
+- start playing music by providing the voice command as text (Works with all music providers)
+
+Some ideas what you can do in your home by using rules and other openHAB controlled devices:
+
+- Automatic turn on your amplifier and connect echo with bluetooth if the echo plays music
+- If the amplifier was turned of, the echo stop playing and disconnect the bluetooth
+- The echo starts playing radio if the light was turned on
+- The echo starts playing radio at specified time 
+- Remind you with a voice message, that a window is open for a long time and it is winter
+- Start a routine which welcome you, if you come home
+- Start a routine which switch a smart home device connected to alexa
+- Start your briefing if you turn on the light first time in the morning
+- Have different flash briefing in the morning and evening
+
+## Note ##
+
+This binding uses the same API as the Web-Browser-Based Alexa site (alexa.amazon.de). In other words, it simulates a user which is using the web page.
```

Comment:
I'am not sure if I understand you correct. Because if I add an line break after each sentence, this will be shown to the user. And other readme does not do this.